### PR TITLE
Add `get_tag_info` to operational message store

### PR DESCRIPTION
**Pull request by JayH5**
`get_tag_info` is used in some operational cases

### CODE REVIEW

**Comment by jerith** [comment]
:+1:

File: vumi_message_store/tests/test_message_store.py [hunk 1]
```diff
@@ -644,6 +644,41 @@ def test_get_event_missing(self):
...
+    def test_get_tag_info_tuple_form(self):
+        """
+        We can specify the tag as a (pool, tagname) tuple.
+        """
+        # We use the internals of the backend here because there's no other
+        # direct way to create a CurrentTag object.
+        stored_tag = self.backend.current_tags(
+            "size:large", current_batch="mybatch")
+        yield stored_tag.save()
+        tag_info = yield self.store.get_tag_info(("size", "large"))
```
Comment (JayH5) [inline vumi_message_store/tests/test_message_store.py:670]: Slightly confused... there's a TODO note on the message store method to allow tuples as the parameter... but we're testing that here?

Comment (jerith) [inline vumi_message_store/tests/test_message_store.py:670]: Good catch. You can remove the `TODO`, but please also add a docstring or comment on the overridden `load` method on the model.

Comment (JayH5) [inline vumi_message_store/tests/test_message_store.py:670]: Done

**Comment by jerith** [comment]
:+1:

**Pull request merged**